packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -50,6 +49,15 @@ contract ChugSplashManager is Owned {
         uint256 total;
     }
 
+    /**
+     * @notice Struct representing a bundle claimed by an executor.
+     */
+    struct ClaimedBundle {
```

Comment:
I think this could be merged into the `ChugSplashBundleState` instead of having a new struct.

Reply:
Avoids introducing a new mapping too.